### DISCUSSION TITLE
How to stop featured artists making their own artist page.

### DISCUSSION THREAD

**Discussion by KaxuTheSheep** [namidaco/namida#419] [Q&A]
In the artists section of the app, some of my songs are by artist I listen to but have featured artist and instead of going into the artist it'll make a new artist called "Artist 1/Artist 2/Artist 3" etc. Is there a way, preferability automatic, which makes these songs not create a new artist but is inside artist 1's songs, artist 2's songs, etc? I'd assume there is some way with the tags however I do not know how and some artist have like 6 artists that they are in

![Screenshot_20241210-165559_1](https://github.com/user-attachments/assets/17f70d4e-42ff-41fa-991a-1ca9b53ea2d9)

**Comment by MSOB7YY** [comment]
by disabling this option from indexer settings
![Screenshot_20241210-190849_Namida](https://github.com/user-attachments/assets/253a495d-4c06-4f68-8b3d-46541e19a9c1)

**Reply by KaxuTheSheep** [reply]
Thank you
